src/components/application_manager/test/commands/mobile/perform_audio_pass_thru_test.cc

```diff
@@ -108,104 +125,503 @@ class PerformAudioPassThruRequestTest
     Mock::VerifyAndClearExpectations(&mock_message_helper_);
   }
 
-  void ResultCommandExpectations(MessageSharedPtr msg,
-                                 const std::string& info) {
+  void ResultCommandInfoExpectations(MessageSharedPtr msg,
+                                     const std::string& info,
+                                     int32_t code,
+                                     bool success) {
     EXPECT_EQ((*msg)[am::strings::msg_params][am::strings::success].asBool(),
-              true);
-    EXPECT_EQ(
-        (*msg)[am::strings::msg_params][am::strings::result_code].asInt(),
-        static_cast<int32_t>(hmi_apis::Common_Result::UNSUPPORTED_RESOURCE));
+              success);
+    EXPECT_EQ((*msg)[am::strings::msg_params][am::strings::result_code].asInt(),
+              static_cast<int32_t>(code));
```

Comment:
@tkireva `code` is already `int32_t` so casting is not needed. Here and in the next method.